test/inductor/test_triton_kernels.py

```diff
@@ -1545,6 +1545,17 @@ def f(x, y):
         x = torch.randn(4, device=GPU_TYPE)
         f(x, x)
 
+    @requires_gpu
+    def test_triton_kernel_float64_constant(self):
+        def f(x):
+            return x * (0.12 * x.shape[0])
+
+        x = torch.ones(200, device=GPU_TYPE, dtype=torch.float64)
```

Comment:
Can you parameterize the dtype here?

Reply:
@isuruf , I don't think so? At least, there's no dtype that I know about. We just have a `sympy.core.numbers.Float` object that we're printing.

Reply:
Oh, misunderstood you comment :/

Reply:
done

Reply:
... except I got it wrong. omg. fix forthcoming